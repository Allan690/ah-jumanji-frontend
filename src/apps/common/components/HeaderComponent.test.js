import React from "react";

import {shallow} from "enzyme";

import Header from "./HeaderComponent";


describe("Header Section", () => {
  const component = shallow(<Header debug label="label" />);

  it("should render correctly ", () => {
    expect(component).toMatchSnapshot();
  });


});
