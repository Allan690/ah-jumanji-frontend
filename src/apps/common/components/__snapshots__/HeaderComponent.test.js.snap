// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Section should render correctly  1`] = `
<div
  className="panel-heading"
  style={
    Object {
      "padding": "20px 20px 0px 20px",
    }
  }
>
  <h3>
    label
  </h3>
  <hr />
</div>
`;
